### PR TITLE
New homepage design

### PR DESCRIPTION
**Pull request by jywarren**
New home page! Solves https://github.com/publiclab/plots2/issues/6155

### CODE REVIEW

**Comment by jywarren** [comment]
Oops, `authors` is defined twice:

```
Method Node#authors is defined at both app/models/node.rb:280 and app/models/node.rb:289. NEW
```

**Comment by jywarren** [comment]
Aiming for:

image

**Comment by jywarren** [comment]
```
ERROR["test_basic_node_attributes", #<Minitest::Reporters::Suite:0x000056377b9b1438 @name="NodeTest">, 15.528653303999988]
 test_basic_node_attributes#NodeTest (15.53s)
NameError:         NameError: undefined local variable or method `first' for #<Node:0x000056377b9e9518>
            app/models/node.rb:317:in `summary'
            test/unit/node_test.rb:19:in `block in <class:NodeTest>'
```

**Comment by jywarren** [comment]
missing.png error could be due to the profile pics in the project... We can make sure a missing.png really exists!

**Comment by jywarren** [comment]
Looking good!

**Comment by jywarren** [comment]
Now we have to decide which people to show under each project, and how to identify them (i.e. how to call them up from the database: tags? profile tags? followers of some tag?)

**Comment by steviepubliclab** [comment]
Would it make sense to show profiles in order of who has tagged most to that particular project?

**Comment by ebarry** [comment]
that sounds great @steviepubliclab ! To my understanding, each project has a unique tag, and people will either have posted content to that tag, or have "joined the group" for that tag which results in a profile tag. Should we get more specific in any way?

**Comment by jywarren** [comment]
OK, so for a project page, do we assume that the name of the page is the same as the project tag? Or should we make that customizable as well?

I'm just thinking -- for the project at https://publiclab.org/wiki/the-sand-sentinel-program for example, the tag isn't `the-sand-sentinel-program`, right? If we do want to be able to customize, we could have a power tag like `project:sand-sentinel` you can add to the project page which would override the default `the-sand-sentinel-program` as the "project's tag"?

**Comment by steviepubliclab** [comment]
Currently, the name of the page isn't always the same as the project tag.

**Comment by jywarren** [comment]
OK, so path forward is:

1. default is that a project called `/wiki/ponyhat`, then we look for contributors to the tag `ponyhat`
2. if a project tag and page URL are different, like `/wiki/bagel` and `breakfast`, we use a special powertag on the wiki like `contributors:breakfast` (maybe?) and then it'll display people who've contributed to that tag instead of to `bagel`

**Comment by SidharthBansal** [comment]
I loved the new homepage design

**Comment by jywarren** [comment]
We now just need to make the lead text area into a feature, then do the people listings follow-ups

File: app/views/home/home.html.erb [hunk 1]
```diff
@@ -1,18 +1,123 @@
-<div class="container">
-
 <% cache('feature_home-intro', skip_digest: true) do %>
   <%= feature("home-intro") %>
```
Comment (jywarren) [inline app/views/home/home.html.erb]: This now needs to be updated with a final version of https://gist.github.com/jywarren/ae51e24b4bcc2242ff2d8083838ce41a once published!

**Comment by jywarren** [comment]
Asked on Slack - 

Should I change that to be `project-featured` or something, so that we have more curatorial control? 

Should it show a random sampling from the project pages? 

Should it only show ones that have a lead image?

**Comment by jywarren** [comment]
🎉

**Pull request merged**